add crd option for FIPS mode (#1893)
DataDog/datadog-operator: docs/configuration.v2alpha1.md

| global.fips.image.pullPolicy | The Kubernetes pull policy for the FIPS sidecar image. Values: Always, Never, IfNotPresent. |
| global.fips.image.pullSecrets | Specifies Docker registry credentials (https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod) for the FIPS sidecar. |
| global.fips.image.tag | Defines the tag of the FIPS sidecar image. This parameter is used if global.fips.image.name does not correspond to a full image string. |
| global.fips.localAddress | The local IP address of the FIPS sidecar. Default: 127.0.0.1. |

> | global.fips.modeEnabled | Enable the Agent containers in FIPS mode. Unless image overrides are configured, the FIPS version of the Agent images will be used. (Requires Agent and Cluster Agent 7.64.0+.) |

Reviewer: ```suggestion
| global.fips.modeEnabled | Enable the Agent containers in FIPS mode. Unless image overrides are configured, the FIPS version of the Agent images is used. (Requires Agent and Cluster Agent 7.64.0+.) |
```
